local_build.py

```diff
@@ -177,8 +183,8 @@ def build_authorea_latex(localdir, builddir, latex_exec, bibtex_exec, outname,
             elif ls in ('posttitle.tex', 'title.tex', 'preamble.tex', 'header.tex'):
                 pass # skip any that have been processed above
             elif ls in ('abstract.tex'):
-                # add abstract to title content
-                titlecontent.append(r'\begin{abstract}' + get_input_string('abstract', get_in_path(localdir, builddir, pathtype))  + '\end{abstract}')
+                # add abstract to section input
+                sectioninputs.append(get_input_string('abstract', get_in_path(localdir, builddir, pathtype)))
```

Comment:
I think you'll need to be more careful with this.  In at least some situations, authorea articles are not expected to have \begin{abstract} \end{abstract} sections.  So maybe the thing to do is to just read the abstract file into a string, and then look through in for the presence of ``r'\begin{abstract}'`` and ``r'\end{abstract}'``.  If they are present, add it the way you're doing here, but if not, add in the section headers.  I think that will address both use cases.